lsp/ts_ls.lua

```diff
@@ -0,0 +1,78 @@
+---https://github.com/typescript-language-server/typescript-language-server
```

Comment:
the docs should start with `--- @brief`

```suggestion
--- @brief
---
---https://github.com/typescript-language-server/typescript-language-server
```